.github/workflows/deploy-helm-charts.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release Helm Charts
+
+# Only cut new helm charts when:
+# - a new release is published
+# - a maintainer runs the workflow explicitly via "gh workflow run"
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout brupop
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.4.1
```

Comment:
Is the plan to enable GitHub Pages to host the published charts?

Reply:
It'll require some iteration to get it right before running the next release:
- I'm not sure if pages for this repo are turned on. But we can use the `workflow-dispatch` method to launch this workflow out of band of a release.
- We'll need to update release processes to capture updating the helm chart metadata (version etc.) alongside other things (like updating the readme).

But really, in theory, it should _just work_.